src/remote/watcher.js

```diff
@@ -71,7 +70,7 @@ export default class RemoteWatcher {
       await this.pouch.setRemoteSeqAsync(changes.last_seq)
       log.debug({event: 'end'}, 'No more remote changes for now')
     } catch (err) {
-      if (err.message === REVOKED) {
+      if (err.status === 400) {
         throw err
```

Comment:
If we end up testing this in multiple places, maybe we could later extract the `if` + `throw` in some `ensureNotRevoked()` function...